Xamarin.Forms.Core/CornerRadius.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Xamarin.Forms.Core
```

Comment:
we don't use that namespace, we use `Xamarin.Forms`

Reply:
This is important. Changed.